faithcomesbyhearing/dbp-etl-web
# FCBHDBP-371 implement DBPDatabaseCheck

## Description
It has created the new option: Database Check and it will run the lambda identified by environment variable `DATABASECHECK_LAMBDA`. It doesn't need an input parameter.

NOTE: I don't have the credentials to execute the lambda function but I have validated that the new button is invoking the lambdaClient with the `POSTVALIDATE_LAMBDA` value.

This PR is related to dbp-elt PR:
- https://github.com/faithcomesbyhearing/dbp-etl/pull/45

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-371

## How Do I QA This
You should go to etldev website using the next URL: https://etldev.dev.dbt.io/ and you should click the new option `Database Check` and then click on `Run` button

## Screenshot
You can see the screenshot about the UI 
![screenshot-localhost_8082-2022 05 06-16_57_59](https://user-images.githubusercontent.com/73488660/167222095-6fcc35b3-531d-41b2-a2ab-f1091076a3f0.png)
updated.
